scripts/install-lint.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
+VERSION="v1.47.2"
 GOBIN="$(go env GOPATH)/bin"
 
 if [[ ! -x "${GOBIN}/golangci-lint" ]]; then
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "${GOBIN}" v1.46.2
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "${GOBIN}" "${VERSION}"
```

Comment:
After I had the PR up, I saw that CI was using a newer version of `golangci-lint` than I was using locally. I matched the version and fixed the new complaints.